src/parsers/parser.js

```diff
@@ -46,17 +47,17 @@ class Parser {
 
     skipPunctuation (punc) {
         if (this.isNextTokenPunctuation(punc)) this.lexer().next();
-        else this.throwError(this.getGenericErrorMsg(this.getCurrentTokenValue()));
+        else this.throwError(feedbackMessages.getGenericErrorMsg(this.getCurrentTokenValue()));
```

Comment:
since it has been renamed to **genericErrorMsg**, you need to update all occurrences accordingly

Reply:
oops sorry my bad